shared_model/builders/protobuf/builder_templates/block_template.hpp

```diff
@@ -84,8 +84,7 @@ namespace shared_model {
       }
 
      public:
-      TemplateBlockBuilder(const SV &validator = SV())
-          : stateless_validator_(validator){};
+      TemplateBlockBuilder() = default;
```

Comment:
before anybody complain:
1. SV has no virtual methods, so can not be overloaded in derived classes. Thus, there is no need in initializing constructor.
2.  If you want to change SV type (for testing purposes, for example), you just change SV template type.